src/components/Onboarding/index.jsx

```diff
@@ -1,8 +1,26 @@
-import styled from "styled-components";
+import { useEffect, useState } from "react";
 
+import styled from "styled-components";
 import Button from "../shared/Button";
+import ToastPopup from "../shared/Toast";
+import { getAuth } from "../../services/auth";
 
 function Onboarding() {
+  const [isClicked, setIsClicked] = useState(false);
+  const [toast, setToast] = useState(false);
+
+  useEffect(() => {
+    if (isClicked) {
+      getAuth();
+    }
```

Comment:
- `P2`
```suggestion
    if (isClicked) {
      getAuth();
      
      setIsClicked(false);
    }
```

이후 사이드 이펙트를 방지하기 위해 `getAuth()` 호출 후 다시 `isClicked`를 `false`로 설정해주시는 것은 어떨까요!